yaroslav.nazarenko/first/task1/task1.rb

```diff
@@ -0,0 +1,5 @@
+x = Array.new(3) { Array.new(3) { rand (-10..10) } }
+puts "Random array: #{x}"
+x.flatten.each.select { |x| puts x if x > 0 }
+pos_num = x.flatten.each.select(&:positive?).sum
```

Comment:
usage of #each is redundant here.
```suggestion
pos_num = x.flatten.select(&:positive?).sum
```